mol-loader/rdkit_utils.py

```diff
@@ -90,3 +93,82 @@ def rdkit_mol_from_smiles(smiles):
         except ValueError:
             pass
     return mol
+
+
+def rdkit_fps_from_mol(mol):
```

Comment:
The RDKit fingerpirnt is a useful one to have available, but in the interest of having the default fingerprint show people more intuitive results, I'd suggest to defaulting to the Morgan fingerprint with radius 2. The function would look something like:
```python
def rdkit_moranfps_from_mol(mol):
    """
    Generate a string representation of the fingerprint of mol
    :param mol: RDKit Molecule
    :return: String representation of the MFP2 fingerprint of mol, number of bits in the fingerprint
    """
    fps = None
    num_bits = None
    if mol is not None:
        try:
            fp = rdMolDescriptors.GetHashedMorganFingerprint(mol,2,nBits=2048)  # you also need "from rdkit.Chem import rdMolDescriptors" at the top of the file
            num_bits = fp.GetNumOnBits()
            fps = DataStructs.BitVectToFPSText(fp)
        except ValueError:
            pass
    return fps, num_bits
```